forcats.Rmd

```diff
@@ -401,12 +401,12 @@ renamed_factor_clarity_3 <- factor_clarity %>%
                `Very Very Slightly Included`= c("VVS2", "VVS1"),
                `Internally Flawless (best)`="IF")
 ```
-It should be noted, that `fct_recode()` could be easily replaced with `fct_collapse()` in the [2nd example for `fct_recode()`](#recode-2) as `fct_collapse()` also works with single element character vectors. Replacing `fct_collapse()` with `fct_recode()` in the example above on the other hand is not possible, because `fct_recode()` cannot work with multiple element vectors such as `c("SI2", "SI1")`. In this regard, we rather recommend using `fct_collapse()` than `fct_recode()` for combining different levels.
+It should be noted, that `fct_recode()` could be easily replaced with `fct_collapse()` in the [2nd example for `fct_recode()`](#recode-2) as `fct_collapse()` also works with single element character vectors. Replacing `fct_collapse()` with `fct_recode()` on the other hand is not possible, because `fct_recode()` cannot work with multiple element vectors such as `c("SI2", "SI1")`. In this case, we recommend using `fct_collapse()` than `fct_recode()` for combining different levels.
 
 ### Aggregate levels into a lump 
-This topic is related to the rationale behind `fct_collapse()` described in the previous section. In contrast to specifying the levels to be combined explicitly, the forcats package also offers the possibility to lump levels together. That is combining several levels together to a single lump level called `Other` by default  There are several different functions that allow lumping based on different criteria.  
-The functions which is probably easiest to grasp is `fct_lump_n()` which takes an integer value `n` as the second (or the first argument in case of piping). 
-It allows to preserve the `n` most (least) frequent levels for `n>0` (`n<0`). All other levels will be lumped together into one level that has the lowest order and referred to as `Other` by default.
+This topic is related to the rationale behind `fct_collapse()` described in the previous section. In contrast to specifying the levels to be combined explicitly, the forcats package also offers the possibility to lump levels together. "Lumping" is the act of combining several levels in order to create a single lump level called `Other` by default.  There are several different functions that allow lumping based on different criteria.  
+The functions which is probably easiest to understand is `fct_lump_n()` which takes an integer value `n` as the second (or the first argument in case of piping). 
+It allows to preserve the `n` most/least frequent levels for `n>0` / `n<0`. All other levels will be lumped together into one level that has the lowest order and is referred to as `Other` by default.
```

Comment:
I came across the notation using parenthesis when reading some scientific papers and since then perceived it as common practice, but maybe this is domain specific. 
The intention is to combine to two sentences into one like this:

```R
# Normal
The weather is good on a sunny day.
The weather is bad on a rainy day.

# Using parenthesis
The weather is good (bad) on a sunny (rainy) day.
```

Maybe @tillschwoerer could help to clarify this.

Reply:
I would say both is fine, a matter of personal taste. Maybe / is more often used in German, and () more often in English texts?

Reply:
I can live with both approaches as well. I only ask you to make the changes for the whole chapter consistent Bente